.travis.yml

```diff
@@ -1,30 +1,44 @@
-language: python
 dist: xenial
-python:
-    - "3.6"
-    - "3.7"
+sudo: required
 
-before_install:
-  - pip freeze
-  - pip uninstall numpy -y && pip install numpy==1.15.4 # FIXME: Temporary fix as f2py is buggy on numpy 1.16.0
-  - sudo apt-get install gfortran
-  - pip install Sphinx sphinx_bootstrap_theme sphinx-autodoc-typehints m2r cffi
-  - sudo apt install snapd
-  - sudo snap install dotnet-sdk --classic
-
-install:
-  - sphinx-build docs/sphinx build/html
-  - touch build/html/.nojekyll # https://help.github.com/articles/files-that-start-with-an-underscore-are-missing/
-  - cd tests/js && npm install
-  - cd $TRAVIS_BUILD_DIR
-
-script:
-  # See https://stackoverflow.com/a/34140498 for why "python -m" is needed.
-  - python3 -m pytest -v -s
-  - cd tests/js && npm test
-  - cd $TRAVIS_BUILD_DIR
-  - cd src/c_sharp && dotnet-sdk.dotnet test
-  - cd $TRAVIS_BUILD_DIR
+matrix:
+  include:
+    - language: python
+      python:
+        - "3.6"
+        - "3.7"
+      before_install:
+        - pip freeze
+        - pip uninstall numpy -y && pip install numpy==1.15.4 # FIXME: Temporary fix as f2py is buggy on numpy 1.16.0
+        - sudo apt-get install gfortran
+        - pip install Sphinx sphinx_bootstrap_theme sphinx-autodoc-typehints m2r cffi
+        - sudo apt install snapd
+        - sudo snap install dotnet-sdk --classic
+      install:
+        - sphinx-build docs/sphinx build/html
+        - touch build/html/.nojekyll # https://help.github.com/articles/files-that-start-with-an-underscore-are-missing/
+        - cd tests/js && npm install
+        - cd $TRAVIS_BUILD_DIR
+      script:
+        # See https://stackoverflow.com/a/34140498 for why "python -m" is needed.
+        - python3 -m pytest -v -s
+        - cd tests/js && npm test
+        - cd $TRAVIS_BUILD_DIR
+        - cd src/c_sharp && dotnet-sdk.dotnet test
+        - cd $TRAVIS_BUILD_DIR
+    - language: r
+      r:
+        - release
+        - devel
+      before_install:
+        - R -e 'install.packages("devtools", dependencies = TRUE)'
+        - cd $TRAVIS_BUILD_DIR/src/r
+        - ./build_package.sh
+      script:
```

Comment:
We need to add tests

Reply:
No need here. The tests are run in `script` stage.

Reply:
`script` is what I meant and there are no tests there -- can you please add tests for the CI as well?

Reply:
Actually the CI tests are run through `R CMD check`. It will untar the source package and run all tests under `tests` folder.

```
      script:
        - R CMD build .
        - R CMD check psychrolib*tar.gz
```

Reply:
Cool -- then this is good as it is! 👍

Reply:
I looked at this again but could not find any info re the tests in the output (https://travis-ci.com/psychrometrics/psychrolib/jobs/273925520#L1303) -- is there any chance you can make this a bit more verbose so that we can understand what actually goes on. To see an example of what I mean, please see outputs from the tests we carry out already at https://travis-ci.com/psychrometrics/psychrolib/jobs/273925520#L1303

Reply:
Sorry, but I did not get your point.
here is an example of current R tests: https://travis-ci.com/psychrometrics/psychrolib/jobs/277571179

I believe it should be enough verbose...

Reply:
Well the only thng it says is `Running ‘testthat.R` `OK` -- (https://travis-ci.com/psychrometrics/psychrolib/jobs/277571179#L7522). It does not tell me anything about which functions are testes etc like we have for other langs. But OK, if there is no easy way, leave it like that.

Reply:
You're right. That is because `R CMD check` runs dozens of checks, instead of only running tests under `tests` folder. But it will print the location of errors if occur. Anyway, I will just leave it as it is here.